src/Voron/Impl/FileHeaders/FileHeader.cs

```diff
@@ -66,17 +72,25 @@ public struct FileHeader
         public int PageSize;
 
         /// <summary>
-        /// Hash of the header used for validation
+        /// The journal id for all the transactions in shared journals
+        /// for this environment that allows to tell which transactions
+        /// belong to this envionment or to others
         /// </summary>
         [FieldOffset(154)]
+        public Guid JournalId;
```

Comment:
is this a concern when we soft-delete a database and create it again? I think it should not matter, but writing that comment here anyway

Reply:
No, that is strictly internal value, only used for recovery details, see:

https://ayende.com/blog/201923-B/accidenal-complecity-a-tale-of-two-guids?key=28ec9f477f3d4cb1869aa362c8def5bf